src/SIL.XForge.Scripture/ClientApp/src/app/translate/training-progress/training-progress.component.ts

```diff
@@ -87,6 +91,17 @@ export class TrainingProgressComponent extends DataLoadingComponent implements O
     this.trainingProgressClosed = true;
   }
 
+  /**
+   * Determines whether the user has the right to edit texts generally, without considering permissions on this chapter.
```

Comment:
I'm not sure this comment makes sense in the context of this component, which isn't concerned with any particular chapter.

Reply:
You are right - fixed.